src/main/java/org/generationcp/middleware/domain/inventory/manager/LotItemDto.java

```diff
@@ -9,8 +9,14 @@
 
 	private Integer gid;
 
+	private Integer storageLocationId;
+
+	/** Ignored if storageLocationId is present */
 	private String storageLocationAbbr;
 
+	private Integer scaleId;
+
+	/** Ignored if scaleId is present */
```

Comment:
This one should be unitName instead to be consistent with the naming we decided to use, but I think we will have to change it in Inventory Manager. We can tackle this one later if it adds extra work